.github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md

```diff
@@ -0,0 +1,54 @@
+---
+name:  Dependencies Issue
+about: Use this template for all Dependencies Issues, even those automatically generated
+labels: dependencies
+---
+
+# Description
+- Information about the dependency and its role in the project
+- Current version - what are we running
+- Target Version - what are we changing it to
+- DependaBot PR link if available
+
+## Features and fixes available
+- List of important bug fixes, features or security patches
+- List of the important changes in this version that could affect the workflow
+
+## Additional information
+- Additional information about the version update.
+- Guideline that can be followed for the process
+
+## QA Needs
+< Indicate whether this item needs QA testing. For those that do, indicate any special setup or tool set the QA team might need. >
+
+## Acceptance Criteria
+< list of behaviors that must be present in order to satisfy the required functionality described above. >
+- [ ] Dependency updated in the build
+- [ ] Unit tests passing
+
+## Announcement Message
+devchat: < Dependency name > has been updated to <dependency target version> See <link to new features or this Issue > for available features and fixes
```

Comment:
Are we sure we don't want a release notes here? most of the dependency updates will effect logic on the back end or UI

Reply:
But if implemented correctly, they should have no visible changes in the UI and no change to the actual business part of the logic, correct? If so, there is nothing to announce to a user. They just don't see it.

Reply:
Ideally, but when reporting bugs if there is a release note pertaining to the things that could be affected by the update. E.g. js-datepicker "Please let us know if you see any problems with the datepicker when selecting orders, or if a customer has issues picking the date of there lab visit for expediting orders" this would prevent bugs from being made, and paying out bounties.

Reply:
got it. Added releasenotes instructions